step-api/src/main/java/org/jenkinsci/plugins/workflow/structs/DescribableHelper.java

```diff
@@ -419,6 +434,7 @@ public Schema getType() {
         private final Class<?> supertype;
         private final Map<String,Schema> types;
         HeterogeneousObjectType(Class<?> supertype, Map<String,Schema> types) {
+            super(supertype);
             this.supertype = supertype;
```

Comment:
Not sure what you mean - whether we're using `Type` or `Class` for the `actualClass`, it still needed to be a new variable in `ParameterType`...